index.html

```diff
@@ -272,6 +272,78 @@ <h3>Virtual Reality Gaming</h3>
 References:
 <a href="https://lists.w3.org/Archives/Public/public-webrtc/2018May/0063.html">Mailing list discussion</a>
 </section>
+<section id="no-trust-webex">
+<h3>Don't Pown My Video Conferencing </h3>
+
+<p>Cloud video conferencing systems have no need to be able to access
+  the cleartext media and text flowing through their servers.
+  Some of these conferencing services desire to be able to promote trust
+  by explicitly showing they do not have access to contents of their
+  users' calls. They are trusted to connect the right people to the
+  conference and to route the packets but they are not trusted to access
+  the audio and video media or text in the call.</p>
+
+<p>Solutions to this problem fall into two major categories: one where
+  the JavaScript comes from a source trusted to see the media contents,
+  and one where it does not:</p>
+
+<h4>Untrusted JavaScript Cloud Conferencing</h4>
+
+<p>There are many cases where a system such as WebEx is trusted to
+  connect the members of a conference but has no need to access the
+  contents of the conference. This is true of the majority of
+  conferencing systems on the web today.  Just to highlight the
+  scope of this requirement, there are more minutes of WebRTC
+  that are used in conferences where the servers have no need
+  to access the contents (e.g. where audio is forwarded rather than
+  mixed) than any other use of WebRTC audio by orders of magnitude.
+  This is one of the primary use case for WebRTC audio and accounts
+  for billions of minutes per month of potential use of WebRTC.
+</p>
+
+<p>In this use case, the JavaScript comes from the operator of the
+  conference bridge. The isolated media features of WebRTC prevent
```

Comment:
Nit: "The isolated media features of WebRTC can prevent the Javascript.."

Reply:
Fixed.